packages/dds/map/src/directory.ts

```diff
@@ -664,12 +664,6 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
         subdirsToRegisterFrom.push(this.root);
```

Comment:
I'm not sure why you did not delete the whole thing - this function is now noop but spends CPU cycles for nothing